common/utils/src/main/scala/org/apache/spark/internal/LogKey.scala

```diff
@@ -26,13 +26,16 @@ trait LogKey {
 }
 
 /**
- * Various keys used for mapped diagnostic contexts(MDC) in logging.
- * All structured logging keys should be defined here for standardization.
+ * Various keys used for mapped diagnostic contexts(MDC) in logging. All structured logging keys
```

Comment:
Many of these formatting changes happened after running dev/scalafmt. Should we keep them?